### ISSUE TITLE
Exception in IntelliJ 2019.3.3

### ISSUE THREAD

**Issue by Davoleo** [wakatime/jetbrains-wakatime#133]
I got it while Opening the IDE

Stacktrace:
```
java.lang.UnsupportedOperationException
	at java.base/java.nio.file.Files.setPosixFilePermissions(Files.java:2078)
	at com.wakatime.intellij.plugin.Dependencies.makeExecutable(Dependencies.java:318)
	at com.wakatime.intellij.plugin.Dependencies.installCLI(Dependencies.java:139)
	at com.wakatime.intellij.plugin.WakaTime$1.run(WakaTime.java:96)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by alanhamlett** [comment]
Which operating system are you using?

**Comment by Davoleo** [comment]
I'm on Windows 10
![image](https://user-images.githubusercontent.com/24458276/75118461-aefc7180-567a-11ea-9422-937d7a9d2b24.png)

**Comment by webpublishers** [comment]
Got exactly the same issue in Webstorm using Windows 10

**Comment by alanhamlett** [comment]
Oops, setting file permissions should have only been running on Linux and Mac but the logic was accidentally inverted. Fixed with 1a740a294656e40b18c695b86795149713e817d1, and releasing a new version of the plugin soon.

**Comment by alanhamlett** [comment]
Fixed in v11.0.0.
